tests/queries/0_stateless/02841_group_array_sorted.sql

```diff
@@ -0,0 +1,22 @@
+SELECT groupArraySorted(5)(number) from numbers(100);
+
+SELECT groupArraySorted(number) from numbers(100);
+
+SELECT groupArraySorted(100)(number) from numbers(1000);
```

Comment:
This test SELECT is conceptually equivalent the first SELECT.

Reply:
The problem persists. If you like to test something useful in l. 3, then check the case that the parameter provided to `groupArraySorted` (= 100) is bigger than the number of input rows.